src/structs.rs

```diff
@@ -1,5 +1,6 @@
 use pyo3::PyErr;
 
+#[derive(Debug)]
```

Comment:
Ha it's funny because I had added this too and didn't end up committing it, I am sure we'll need it.